packaging/plugins/provision.sh

```diff
@@ -124,7 +119,8 @@ function wagon_create_package(){
          popd
     popd
 
-    docker run -v $PLUGIN_PATH:/packaging $IMAGE_NAME
+    docker run --name $CONTAINER_NAME -v $PLUGIN_PATH:/packaging $IMAGE_NAME
+    echo "## echo build wagon package created successfully"
```

Comment:
Well you don't need to write echo in the echo message :)

Reply:
Removed